build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,8 +7,8 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.1.0-preview1-24508-06";
-        public static readonly string JitVersion = "1.1.0-preview1-24508-06";
+        public static readonly string CoreCLRVersion = "1.1.0-preview1-24512-01";
+        public static readonly string JitVersion = "1.1.0-preview1-24512-01";
         public static readonly string CoreCLRVersion_Latest = "1.1.0-preview1-24508-06";
         public static readonly string JitVersion_Latest = "1.1.0-preview1-24508-06";
```

Comment:
Getting these the same is waiting on https://github.com/dotnet/core-setup/pull/250#discussion_r78209267, the plan being to remove the `_Latest` pair.